src/Sentry/HubExtensions.cs

```diff
@@ -51,6 +51,19 @@ public static ITransactionTracer StartTransaction(
         SentryTraceHeader traceHeader) =>
         hub.StartTransaction(new TransactionContext(name, operation, traceHeader));
 
+    /// <summary>
+    /// Starts a span or transaction if a transaction is not already active on the scope
+    /// </summary>
+    public static ISpan StartSpan(this IHub hub, string operation, string description)
```

Comment:
question: Could we keep this method `internal` for now, to not increase the public surface area?

(since the `class Hub` it was moved from is also `internal`)

Reply:
I feel this method is a good thing in general.  I plan to use it externally to Sentry.

Reply:
Yeah I think this is a pretty common operation for SDK users 👍🏻